src/base.config.ts

```diff
@@ -335,12 +378,26 @@ export default function webpackConfigFactory(args: any): WebpackConfiguration {
 				}
 			}),
 			new webpack.NamedChunksPlugin(),
-			new webpack.NamedModulesPlugin(),
 			(args.externals || isTest) &&
 				new WrapperPlugin({
 					test: /(main.*(\.js$))/,
 					footer: `\ntypeof define === 'function' && define.amd && require(['${libraryName}']);`
 				}),
+			new WrapperPlugin({
+				test: /(bootstrap.*(\.js$))/,
+				header: () => {
```

Comment:
be good if we could hoist this to keep the config clean

Reply:
Okay, I'll hoist the plugin.